tests/TestBaseClass.php

```diff
@@ -37,6 +37,15 @@ public static function setUpBeforeClass(): void
     {
         parent::setUpBeforeClass();
 
+        // Enable Debug and Error Reporting if logging is enabled
```

Comment:
Needed here or unrelated ?

Reply:
I was having trouble with a test and implemented this pending artifact for having more info on failing tests. I can split that into a separate PR if needed.

Reply:
For master : remoive it, but if in develop keep it ?